### PR TITLE
yangster: remove 'yangster' image

### PR DESCRIPTION
**Pull request by vince-fugnitto**
#### What it does

Fixes: #388 

The following pull-request includes updates to **yangster**.

The `yangster-docker` image is removed since it makes use of the outdated `yangster` Theia-extension (which was replaced by the [`yang-vscode`](https://github.com/theia-ide/yang-vscode) vscode extension. The `theia-full` image is updated to make use of the extension instead (so there is no need for the individual image anymore).

#### How to test

The `yangster-docker` image is removed so verify that there are no longer any references to `yangster` besides the new vscode extension. The `full-image` should now support the new extension.

The build currently fails due to #386 

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>

### CODE REVIEW

**Comment by marcdumais-work** [comment]
Until https://github.com/eclipse-theia/theia/pull/8315#pullrequestreview-462477628 is merged and released at the end of the month, the `latest` full image will not have a working yang extension. We could add a quick `chmod` after extensions download, to make `plugins/<yangster>/extension/server/bin/yang-language-server` executable, as a temporary work-around.

**Comment by vince-fugnitto** [comment]
> Until [eclipse-theia/theia#8315 (review)](https://github.com/eclipse-theia/theia/pull/8315#pullrequestreview-462477628) is merged and released at the end of the month, the `latest` full image will not have a working yang extension. We could add a quick `chmod` after extensions download, to make `plugins/<yangster>/extension/server/bin/yang-language-server` executable, as a temporary work-around.

@marcdumais-work we also have the #386 error to worry about before we can officially make use of the extension properly.

File: theia-full-docker/Dockerfile [hunk 1]
```diff
@@ -316,6 +316,9 @@ yarn --cache-folder ./ycache && rm -rf ./ycache && \
      NODE_OPTIONS="--max_old_space_size=4096" yarn theia build && yarn theia download:plugins \
 ;fi
 
+# Change permissions to make the `yang-language-server` executable.
```
Comment (marcdumais-work) [inline theia-full-docker/Dockerfile:319]: Paul's fix has been merged. We'll still need this work-around for `latest` until end of month.

**Comment by marcdumais-work** [comment]
I think `theia-full@next` is failing because of `@theia/rust`, which you take case-of in your other PR.

**Pull request merged**